WordPress/gutenberg
# Add boolean contentStyle and clientId check to Column Edit InnerBlocks

## What?
This PR adds a boolean check to values passed to the `<InnerBlocks />` component of [block-library/src/column/edit.native.js](https://github.com/WordPress/gutenberg/blob/33d84b036592a5bf31af05b7710f3b2b14163dc4/packages/block-library/src/column/edit.native.js).

Related items:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5403
- https://github.com/wordpress-mobile/gutenberg-mobile/issues/5387

## Why?
This is an attempt to provide a fix for https://github.com/wordpress-mobile/gutenberg-mobile/issues/5387, which investigates an issue reported via Sentry: `RCTFatalException: Unhandled JS Exception: TypeError: undefined is not an object (evaluating 'b[W].width')`.

Currently, we have not found a way to replicate this issue in production. Based on the nature of the stack trace, it's possible that a width value may be `null` or `undefined` when being passed to `<InnerBlocks />`, and is possibly an unintended side-effect of internal React Native changes made in the [upgrade](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5193) between 0.66.2 and 0.69.4. Until a more specific resolution can be found, this fix is proposed as the user impact rate of this crash is currently 0.098%. The intent is that the boolean checks may prevent `null` or `undefined` values from being passed to other components, and if that is true, we would see the user impact threshold decrease in Sentry as evidence. 

## How?
Checks for both the value of `contentStyle` and `contentStyle[ clientId ]` before passing the `parentWidth` value to the component:

```
<InnerBlocks
    ...
	parentWidth={
		contentStyle &&
		contentStyle[ clientId ] &&
		contentStyle[ clientId ].width
	}    ...
/>
```

## Testing Instructions
1. Create a post
2. Add a Column block
3. Adjust the base settings of the Column layout in any way you prefer
4. Tap the gear icon to further edit Column settings as you prefer
5. App should not crash
